docs/getting_started/trivia_template.md

```diff
@@ -10,28 +10,28 @@ For starters, we recommend that you start by cloning a bot template and customiz
 
 ## The **Trivia Template**
 
-For the rest of this entire guide, we will be customizing a simple template that we created specially for you, learner of Botpress! We intentionally left out many details and elements that we will be adding together during this guide.
+In this guide, we will be customizing a simple template of the Trivia bot that we created specially for you, the learner of Botpress! We intentionally left out many details and elements from the Trivia bot that you will be able to add by following this guide.
```

Comment:
This phrase doesn't make sense to me, the template *is* the trivia bot that we created